searchconsole/query.py

```diff
@@ -181,26 +181,26 @@ def search_type(self, search_type):
     @utils.immutable
     def data_state(self, data_state):
         """
-        Return a new query filtered by the specified dataState, which allows you 
+        Return a new query filtered by the specified data_state, which allows you 
         to include fresh (not finalized) data in your API call.  
 
         Fresh data: data as recent as less than a day old. Fresh data point can 
         be replaced with the final data point after a few days. 
 
         Args:
-            dataState (str): The dataState you would like to use for your report. 
+            data_state (str): The data_state you would like to use for your report. 
                 Possible values: 'final' (default - only finalized data), 
                 'all' (finalized & fresh data).
 
         Returns:
             `searchconsole.query.Query`
 
         Usage:
-            >>> query.dataState('final')
+            >>> query.data_state('final')
             <searchconsole.query.Query(...)>
         """
 
-        self.raw['dataState'] = data_state
+        self.raw['data_state'] = data_state
```

Comment:
Careful - the value in the `raw` dictionary still needs to `dataState` because that's actually what's going in the API request. Make sense?

Reply:
Fixed!